src/pybind/mgr/dashboard/controllers/orchestrator.py

```diff
@@ -102,10 +102,13 @@ def identify_device(self, hostname, device, duration):
 class OrchestratorInventory(RESTController):
 
     @raise_if_no_orchestrator
-    def list(self, hostname=None):
+    def list(self, hostname=None, refresh=None):
```

Comment:
Note that having a `refresh=True` for cephadm is not sustainable as it blocks everything and you cannot cancel it. My thought is to completely drop support for  `refresh=True` in cephadm and raise an exception instead.